ui/nginx.conf

```diff
@@ -0,0 +1,11 @@
+server {
+    listen 80;
```

Comment:
I think the port should be either 8080 or the value passed by PORT environment variables!

Ref: [Listening for requests on the correct port (services), Google Cloud](https://cloud.google.com/run/docs/container-contract#port)